beetime/settings.py

```diff
@@ -1,85 +1,33 @@
-from settings_layout import Ui_BeeminderSettings
+import json
+import os
+
+from PyQt5.QtWidgets import *
 
 from aqt import mw
 from aqt.qt import *
 
-from sync import BEE
+from beetime.settings_layout import Ui_BeeminderSettings
+
+BEE = 'bee_conf'  # Name of key in anki configuration dict
+CONFIG_FILE = os.path.join(os.path.dirname(__file__), 'resources', 'config.json')
```

Comment:
https://apps.ankiweb.net/docs/addons.html#configuration anki has support for this now btw

Reply:
Thanks for the suggestion. I refactored the code and also decided to make use of this.